### PR TITLE
adding updated tooltip files

### PR DESCRIPTION
**Pull request by milanzmitrovic**

### CODE REVIEW

File: examples/tooltip.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+import dash_mantine_components as dmc
+
+
+pio.renderers.default = 'browser'
```
Comment (AnnMarieW) [inline examples/tooltip.py:9]: What's this for?

File: examples/tooltip.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+fig.update_layout(margin=dict(t=50, l=25, r=25, b=25))
+
+# turn off native plotly.js hover effects - make sure to use
+# hoverinfo="none" rather than "skip" which also halts events.
+fig.update_traces(hoverinfo="none", hovertemplate=None)
+
+fig.update_layout(
+    xaxis=dict(title='Year'),
+    yaxis=dict(title='Population'),
```
Comment (AnnMarieW) [inline examples/tooltip.py:51]: You mean "Profit" ?
Actually the default X and Y titles of "Order Date" and "Profit" are pretty good.

File: examples/tooltip.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    plot_bgcolor='rgba(255,255,255,0.1)'
+)
+
+app = Dash(__name__)
+
+app.layout = dmc.Container([
+    html.H4('Dash Tooltip Example', style={'text-align' :'center'}),
```
Comment (AnnMarieW) [inline examples/tooltip.py:58]: Should be camel case `"textAlign` otherwise there are errors in the console.

File: examples/tooltip.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    Output("tooltip-graph", "children"),
+    Input("tooltip-graph-basic-2", "hoverData"),
+)
+def display_hover(hover_data):
+
+    # Do not show tooltip popup window if mouse is not hovered over
+    if hover_data is None:
+        return False, no_update, no_update
+
+    print(hover_data)
```
Comment (AnnMarieW) [inline examples/tooltip.py:81]: Should remove the print statement for production.  Or keep it as a comment line to show how to debug?

File: examples/tooltip.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+    children = [
+        html.Div([
+
+            html.H4(f"Year-Month: {date_month}"),
+            dcc.Graph(figure=fig1),
+
+        ], style={'width': '200px',
+                  'height': '200px',
+                  'white-space': 'normal'
```
Comment (AnnMarieW) [inline examples/tooltip.py:120]: Should be "whiteSpace"

**Review by AnnMarieW — COMMENTED** [review]
Hi Milan.  This is a great example!  Normally I don't like to have a lot of comments but in this app, they are very helpful and well done.

File: examples/tooltip.py (hunk 1 above)
Comment (milanzmitrovic) [reply examples/tooltip.py]: Not necessary here. Just removed.

I use it for local development when I have interactive session.

**Comment by milanzmitrovic** [comment]
Thanks @AnnMarieW for giving me feedback! :)

File: examples/tooltip.py [hunk 6]
```diff
@@ -0,0 +1,124 @@
+
+from dash import Dash, dcc, html, Input, Output, no_update
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
```
Comment (Coding-with-Adam) [inline examples/tooltip.py:5]:
```suggestion
```

**Review by Coding-with-Adam — APPROVED** [review]
Thank you @milanzmitrovic Looks great. I really like the educational comments that explain to the user the more complicated parts of the code.

**Comment by Coding-with-Adam** [comment]
Thank you for your help @AnnMarieW . I ended up commenting out the print statement.

**Comment by AnnMarieW** [comment]
>When running inside the dash-example-index, the bar graph shrinks and it look like it's only 50 pixels wide. It's hard to see, so I updated the code from dmc.Container to dbc.Container

@Coding-with-Adam   @milanzmitrovic   That's odd - I tried adding `fluid=True` to the `dmc.Container` but that didn't fix it either.  Instead of changing to `dash-bootstrap-components` how about just using a `html.Div`  since this app only uses one component the `dmc.Container()` or `dbc.Container()`    Then we can eliminate an import statement.

**Comment by Coding-with-Adam** [comment]
Good idea, @AnnMarieW .
I changed it to `html.Div`. No need for dbc or dmc in this app. It looks good.

**Pull request merged**